src/LaunchDarkly/Server/Client.hs

```diff
@@ -266,11 +266,15 @@ identify :: Client -> Context -> IO ()
 identify client (Invalid err) = clientRunLogger client $ $(logWarn) $ "identify called with an invalid context: " <> err
 identify client context = case (getValue "key" context) of
     (String "") -> clientRunLogger client $ $(logWarn) "identify called with empty key"
-    _ -> do
-        let redacted = redactContext (getField @"config" client) context
-        x <- makeBaseEvent $ IdentifyEvent {key = getKey context, context = redacted}
-        _ <- noticeContext (getField @"events" client) context
-        queueEvent (getField @"config" client) (getField @"events" client) (EventTypeIdentify x)
+    _anyValidKey -> do
+        let identifyContext = optionallyRedactAnonymous (getField @"config" client) context
+        case identifyContext of
+            (Invalid err) -> clientRunLogger client $ $(logWarn) $ "identify called with an invalid context: " <> err
```

Comment:
Are we going to log this whenever a single anonymous context is identified, or when a multi-context composed of only anonymous contexts is identified?

Reply:
Good point. This probably shouldn't log at all since it doesn't really matter.